examples/edit_box.rb

```diff
@@ -0,0 +1,18 @@
+Shoes.app(width:2000 , height:1000) do 
+    edit_box "Heyyya" 
+    para "hi there"
+
+    edit_box "cool but i am bigger and better!!", width: 500, height: 300,wrap_mode: :none
+    para "    "
+    edit_box "okay but i have good wrapping!! unlike u 🫢", width: 100, height: 300,wrap_mode: :word
+    para "   "
```

Comment:
these para are just for spaces.. as ig textview by default dont have any space/margin arnd it